fcrepo-client-impl/pom.xml

```diff
@@ -78,5 +82,37 @@
       <groupId>org.fcrepo</groupId>
       <artifactId>fcrepo-kernel</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.fcrepo</groupId>
+      <artifactId>fcrepo-webapp</artifactId>
+      <version>4.1.2-SNAPSHOT</version>
```

Comment:
Can we move this version number up into a property? or better yet, align it with the version number of fcrepo4-client.